### PR TITLE
Added event formatting helper and changed l2t CSV to use DSV base class

### PR DESCRIPTION
**Pull request by joachimmetz**
Added event formatting helper and changed l2t CSV to use DSV base class

### CODE REVIEW

File: plaso/output/dynamic.py [hunk 1]
```diff
@@ -259,14 +153,14 @@ class DynamicOutputModule(shared_dsv.DSVOutputModule):
       'message', 'parser', 'display_name', 'tag']
 
   def __init__(self, output_mediator):
-    """Initializes an output module object.
+    """Initializes a dynamic delimeter separated values output module object.
```
Comment (Onager) [inline plaso/output/dynamic.py:156]: This is a typo, the word is "delimiter" not "delimeter". Please fix the other instances of this as well, it seems this was missed in the past.

Comment (joachimmetz) [reply plaso/output/dynamic.py:156]: ack, replace 4 instance across the entire code base

**Comment by joachimmetz** [comment]
@Onager PR rebased and review comments addressed, PTAL

**Comment by joachimmetz** [comment]
Rebased and addressed merge issues.

**Pull request merged**